pepper-apis/dss-database/src/main/resources/db-changes/schema/DDP-8248.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                   http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+    <changeSet author="Dmitrii Eliseev" id="2022-08-08-file-upload-notification">
+        <addColumn tableName="file_upload">
+            <column name="notification_sent_at" type="datetime(6)">
+                <constraints nullable="true"/>
+            </column>
+        </addColumn>
+
+        <addColumn tableName="umbrella_study">
+            <column name="notification_email" type="text" defaultValue="null">
```

Comment:
I don't seem to find my comments I made yesterday on this new column ! Adding again.
Did we consider using existing column study_email which is usually a distribution list of study staff.

How is this column getting populated ? I don't see changes to StudyBuilder:getStudyOrInsert:JdbiUmbrellaStudy:insert
Since singular is existing study that's already in prod, might need a patch to populate this new column value !

Reply:
My answers were:
1. Filling this column with the e-mail is a part of the instruction that Blake prepared for release.
As for the study_email column, I use it as the "from" field. I'm not sure we should have one e-mail address for file uploads and other notifications.

2. The column is nullable with a default null value. So, no need to update the insert statement.